pkg/util/mount/mount.go

```diff
@@ -150,6 +158,30 @@ type MountPoint struct {
 	Pass   int
 }
 
+// This represents a single line in /proc/<pid>/mountinfo.
+type MountInfo struct {
+	// Unique ID for the mount (maybe reused after umount).
+	ID int
+	// The ID of the parent mount (or of self for the root of this mount namespace's mount tree).
+	ParentID int
+	// The value of `st_dev` for files on this filesystem.
+	MajorMinor string
+	// The pathname of the directory in the filesystem which forms the root of this mount.
+	Root string
+	// Mount source, filesystem-specific information. e.g. device, tmpfs name.
+	Source string
+	// Mount point, the pathname of the mount point.
+	Path string
+	// Optional fieds, zero or more fields of the form "tag[:value]".
+	OptionalFields []string
+	// The filesystem type in the form "type[.subtype]".
+	Type string
+	// Per-mount options.
+	Opts []string
+	// Per-superblock options.
+	SuperOpts []string
```

Comment:
Do we still need this?  Is this going to break anybody that was expecting the option to be in Opts?

Reply:
`MountInfo` is a separate struct now, I think anybody trying to use it needs to know what fields are first, it will not break anybody IMO.

`Opts` and `SuperOpts` are not used currently. I parsed all fields out from `mountinfo` according to [its doc](http://man7.org/linux/man-pages/man5/proc.5.html) in case future needs arise.

If you prefer, I can remove this fields. What do you think?